mindsdb/integrations/handlers/gitlab_handler/gitlab_tables.py

```diff
@@ -0,0 +1,191 @@
+import pandas as pd
+
+from typing import List
+
+from mindsdb.integrations.libs.api_handler import APITable
+from mindsdb.integrations.utilities.sql_utils import extract_comparison_conditions
+from mindsdb.utilities.log import get_log
+
+from mindsdb_sql.parser import ast
+
+logger = get_log("integrations.gitlab_handler")
+
+class GitlabIssuesTable(APITable):
+    """The GitLab Issue Table implementation"""
+
+    def select(self, query: ast.Select) -> pd.DataFrame:
+        """Pulls data from the GitLab "List repository issues" API
+        Args:
+            query: SELECT
+        Returns:
+            DataFrame
+        Raises:
+            ValueError
+        """
+
+        conditions = extract_comparison_conditions(query.where)
+
+        if query.limit:
+            total_results = query.limit.value
+        else:
+            total_results = 20
+
+        issues_kwargs = {}
+        order_by_conditions = {}
+
+        if query.order_by and len(query.order_by) > 0:
+            order_by_conditions["columns"] = []
+            order_by_conditions["ascending"] = []
+
+            for an_order in query.order_by:
+                if an_order.field.parts[0] != "issues":
+                    next
+
+                if an_order.field.parts[1] in ["created", "updated"]:
+                    if issues_kwargs != {}:
+                        raise ValueError(
+                            "Duplicate order conditions found for created/updated"
+                        )
+
+                    issues_kwargs["sort"] = an_order.field.parts[1]
+                    issues_kwargs["direction"] = an_order.direction
+                elif an_order.field.parts[1] in self.get_columns():
+                    order_by_conditions["columns"].append(an_order.field.parts[1])
+
+                    if an_order.direction == "ASC":
+                        order_by_conditions["ascending"].append(True)
+                    else:
+                        order_by_conditions["ascending"].append(False)
```

Comment:
Can't find where `order_by_conditions` is used for applying sorting or conditions

Reply:
Apply order by condition to the select query